bears/documentation/DocumentationStyleBear.py

```diff
@@ -82,9 +85,14 @@ def process_documentation(self,
                 if stripped_desc[0] != '':
                     stripped_desc.insert(0, '')
 
-            # Indent with 4 spaces.
-            stripped_desc = ('' if line == '' else ' ' * indent_size
-                             + line for line in stripped_desc)
+            # Indent with 4 spaces or a tab.
+            if use_spaces:
+                stripped_desc = ('' if line == '' else ' ' * indent_size +
+                                 line for line in stripped_desc)
+            else:
+                indent_size = 1
```

Comment:
what if I want to have 2 tabs of indentation? I wouldn't overwrite `indent_size`. Also that would be something that should have been mentioned in the docs :+1: